website/src/pages/roadmap.md

```diff
@@ -4,69 +4,55 @@ last_modified_at: 2019-12-30T15:59:57-04:00
-|[1.0.0](https://issues.apache.org/jira/projects/HUDI/versions/12351210)|Summer 2022|
+| Release                                                              | Timeline       |
+|----------------------------------------------------------------------|----------------|
+| [0.12.1](https://issues.apache.org/jira/projects/HUDI/versions/12352182) | Early Q4 2022  |
+| [0.13.0](https://issues.apache.org/jira/projects/HUDI/versions/12352101) | Mid Q4 2022    |
+| [0.13.1](https://issues.apache.org/jira/projects/HUDI/versions/12352250) | TBD            |
 
-## Transactions/Database Layer
+## Transactional Database Layer
 
 |Feature|Target Release|Tracking|
 |------------|--------|-----------|
-|Space-filling curves hardening & perf improvements |0.11|[HUDI-2100](https://issues.apache.org/jira/browse/HUDI-2100)|
-|Metadata table update via multi-table transactions, turned on by default |0.11|[HUDI-1292](https://issues.apache.org/jira/browse/HUDI-1292)|
-|Metadata Index, as a bloom index alternative, fetching col_stats and bloom_filters from metadata table, improving upsert performance. |0.11|[HUDI-1822](https://issues.apache.org/jira/browse/HUDI-1822), [RFC-37](https://github.com/apache/hudi/pull/3989)|
-|Support for Encryption |0.11|[HUDI-2370](https://issues.apache.org/jira/browse/HUDI-2370)|
-|Schema-on-read for non-backwards compatible schema evolution |0.11|[HUDI-2429](https://issues.apache.org/jira/browse/HUDI-2429)|
-|Improvements to merge-on-read log merging/reading with streaming semantics |0.11|[HUDI-3081](https://issues.apache.org/jira/browse/HUDI-3081)|
-|Indexed columns support & elimination of partitioning |0.11|[HUDI-512](https://issues.apache.org/jira/browse/HUDI-512)|
-|Record-level index to speed up uuid based upserts/deletes |0.12|[HUDI-53](https://issues.apache.org/jira/browse/HUDI-53)|
-|Eager conflict detection for Optimistic Concurrency Control |0.12|[HUDI-1575](https://issues.apache.org/jira/browse/HUDI-1575)|
-|Indexed timeline and infinite retention of versions |0.12|RFC coming soon|
-|Improvements to streaming read and full CDC data model support |0.12| [HUDI-2749](https://issues.apache.org/jira/browse/HUDI-2749), RFC coming soon|
-|Consistent hashing based file distribution over storage to overcome throttling issues for very large tables |0.12|RFC published soon|
-|Lock free concurrency control |0.12 -> 1.0.0|[HUDI-3187](https://issues.apache.org/jira/browse/HUDI-3187)|
-|Non-blocking/Lock-free updates during clustering |0.12 -> 1.0.0|[HUDI-1042](https://issues.apache.org/jira/browse/HUDI-1042)|
-|Time Travel updates, deletes |0.12 -> 1.0.0 ||
-|General purpose support for multi-table transactions |0.12 -> 1.0.0||
+| Metaserver for all metadata | 0.13 | [Strawman design](https://docs.google.com/presentation/d/1QBgLw11TM2Qf1KUESofGrQDb63EuggNCpPaxc82Kldo/edit#slide=id.gf7e0551254_0_5), [RFC-36](https://github.com/apache/hudi/pull/4718), [HUDI-3345](https://issues.apache.org/jira/browse/HUDI-3345) |
```

Comment:
the Strawman design doc here is pointing to a caching design. is it a typo?

Reply:
Good catch!  I fixed the link.